components/helm-broker/internal/bundle/syncer.go

```
@@ -59,17 +69,29 @@ func (s *Syncer) Execute() error {
 	s.log.Info("Loading bundles from repositories")
 	defer s.log.Info("Loading bundles finished")
 
-	fetchedBundles := s.fetchBundlesFromRepositories()
-	uniqueBundles := s.removeAndNotifyAboutDuplication(fetchedBundles)
+	previousBundles, err := s.bundleStorage.FindAll()
+	if err != nil {
+		s.log.Errorf("Could not load existing bundles, error: %s", err)
```

why we do not want to return the error?